core/funds.py

```diff
@@ -31,3 +30,20 @@ def search(to_search: dict, word_list: List[str]):
                 break
 
     return return_list
+
+def get_funds(search_items = None):
```

Comment:
🔧 Let's add docstrings to these functions.

Reply:
👌